src/axes/categoryAxis.ts

```diff
@@ -303,28 +324,28 @@ namespace Plottable.Axes {
     public renderImmediately() {
       super.renderImmediately();
       let catScale = <Scales.Category> this._scale;
-      let tickLabels = this._tickLabelContainer.selectAll("." + Axis.TICK_LABEL_CLASS).data(this._scale.domain(), (d) => d);
+      const [downsampledDomain, stepWidth] = this.getDownsampleInfo();
+      let tickLabels = this._tickLabelContainer.selectAll("." + Axis.TICK_LABEL_CLASS).data(downsampledDomain, (d) => d);
+      // Give each tick a stepWidth of space which will partition the entire axis evenly
+      let availableTextWidth = stepWidth;
+      if (this._isHorizontal() && this._tickLabelMaxWidth != null) {
```

Comment:
This variable is named incorrectly; for vertical axes `availableTextWidth` actually represents the vertical height between successive ticks (aka the step width). tickLabelMaxWidth should only affect the width available so when it's vertical, nothing should happen. I've gone and renamed with `availableTextSpace`. I actually also found a subtle bug here, so fixed that too